### PR TITLE
[Keyboard] Add Igloo

### PR DESCRIPTION
**Pull request by djok0127**
Pull request for Igloo

## Description

Adding support for new keyboard igloo

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x]Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/igloo/config.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*
```
Comment (waffle87) [inline keyboards/igloo/config.h:1]: This file should be removed.

File: keyboards/igloo/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Igloo
+
+![Igloo]
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Chris Shin](https://github.com/Chris Shin)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (waffle87) [inline keyboards/igloo/readme.md:9]: These fields should be filled out or removed.

File: keyboards/igloo/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```
Comment (waffle87) [inline keyboards/igloo/readme.md:27]:
```suggestion
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/igloo/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,42 @@
+/*
```
Comment (zvecr) [inline keyboards/igloo/via/keymap.c:1]: This keymap is not inside the keymaps folder

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
As an fyi, you have a bunch of garbage commits pulled in from master. You can see this in the file changed tab and added labels .These will have to be removed for the PR to be considered mergable.

**Comment by djok0127** [comment]
Reverted the merge, sorry about that

**Review by drashna — APPROVED** [review]

File: keyboards/igloo/readme.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Igloo
+
+![Igloo]
+
```
Comment (waffle87) [inline keyboards/igloo/readme.md:4]:
```suggestion
```

**Review by waffle87 — APPROVED** [review]

**Pull request merged**